sys/nginx/default.conf.template

```diff
@@ -66,14 +66,16 @@ server {
     add_header X-XSS-Protection "1; mode=block";
 
     location ~ ^/dark-canary {
+        internal;
+
         set_by_lua_block $request_url {
             return string.gsub(ngx.var.request_uri, "/dark%-canary", "")
         }
         set_by_lua_block $document_url {
             return string.gsub(ngx.var.document_uri, "/dark%-canary", "")
         }
 
-        fastcgi_pass phpfpm-darkcanary:9001;
+        fastcgi_pass ${PHPFPM_CANARY_HOST}:9001;
```

Comment:
This way NGINX will check the resolver on every request .. is this intended?

Reply:
This will be replaced only once when the container starts.

Reply:
uh yep, sorry .. I forgot this is a template :)